src/schema/artist/index.js

```diff
@@ -502,6 +503,7 @@ export const ArtistType = new GraphQLObjectType({
           }).then(shows => showsWithBLacklistedPartnersRemoved(shows))
         },
       },
+      filtered_artworks: filterArtworks("artist_id"),
```

Comment:
This adds the proper filtered artworks schema (w/ connection and aggregations) into the artist schema (presumably we'll be lumping in the filter fragment into all the other artist fragments in our overall SSR Relay query).